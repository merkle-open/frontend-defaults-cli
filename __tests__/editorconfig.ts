import { defaultIt } from './shared';

describe('editorconfig', async () => {
	it('default', async () => await defaultIt('editorconfig', '--editorconfig --force --noInstall'));
});
