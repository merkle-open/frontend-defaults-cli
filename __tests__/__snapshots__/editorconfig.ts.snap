// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`editorconfig default 1`] = `"# EditorConfig. Frontend. Default. Namics.# @see: http://EditorConfig.org# install a plugin to your editor: http://editorconfig.org/#download# mark: not all plugins supports the same EditorConfig properties# This is the top-most .editorconfig file (do not search in parent directories)root = true### All files[*]# Force charset utf-8charset = utf-8# Indentationindent_style = tabindent_size = 4# line breaks and whitespaceinsert_final_newline = truetrim_trailing_whitespace = true# end_of_line = lf### Frontend files[*.{css,scss,less,js,json,ts,sass,php,html,hbs,mustache,phtml,html.twig}]### Markdown[*.md]indent_style = spaceindent_size = 4trim_trailing_whitespace = false### YAML[*.yml]indent_style = spaceindent_size = 2### Specific files[{package,bower,lerna}.json]indent_style = spaceindent_size = 2"`;

exports[`editorconfig default 2`] = `"{  \\"options\\": {    \\"ts\\": false,    \\"tslint\\": false,    \\"es\\": false,    \\"eslint\\": false,    \\"editorconfig\\": true,    \\"prettier\\": false,    \\"stylelint\\": false,    \\"gitignore\\": false,    \\"npmrc\\": false,    \\"readme\\": false,    \\"githooks\\": false,    \\"commitlint\\": false,    \\"nodeVersion\\": false,    \\"webpack\\": false,    \\"build\\": false,    \\"install\\": false,    \\"force\\": true,    \\"dryRun\\": false,    \\"mode\\": \\"api\\"  },  \\"mergedFiles\\": {    \\".editorconfig\\": {      \\"override\\": false,      \\"data\\": \\"# EditorConfig. Frontend. Default. Namics.\\\\n# @see: http://EditorConfig.org\\\\n# install a plugin to your editor: http://editorconfig.org/#download\\\\n# mark: not all plugins supports the same EditorConfig properties\\\\n\\\\n# This is the top-most .editorconfig file (do not search in parent directories)\\\\nroot = true\\\\n\\\\n### All files\\\\n[*]\\\\n# Force charset utf-8\\\\ncharset = utf-8\\\\n# Indentation\\\\nindent_style = tab\\\\nindent_size = 4\\\\n# line breaks and whitespace\\\\ninsert_final_newline = true\\\\ntrim_trailing_whitespace = true\\\\n# end_of_line = lf\\\\n\\\\n### Frontend files\\\\n[*.{css,scss,less,js,json,ts,sass,php,html,hbs,mustache,phtml,html.twig}]\\\\n\\\\n### Markdown\\\\n[*.md]\\\\nindent_style = space\\\\nindent_size = 4\\\\ntrim_trailing_whitespace = false\\\\n\\\\n### YAML\\\\n[*.yml]\\\\nindent_style = space\\\\nindent_size = 2\\\\n\\\\n### Specific files\\\\n[{package,bower,lerna}.json]\\\\nindent_style = space\\\\nindent_size = 2\\\\n\\"    }  }}"`;
